webpack.config.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
   entry: './index.js',
   output: {
-    filename: '[name].min.js',
-    library: 'equivalency',
+    filename: 'equivalency.min.js',
+    libraryTarget: 'commonjs2',
```

Comment:
This results in the built output file having `module.exports=X` where `X` is the default export of the entry point.